hack/verify-licenses.sh

```diff
@@ -128,17 +128,13 @@ do
 			echo "${GO_PACKAGE}  ${LICENSE_NAME}  ${LICENSE_URL}" >> "${KUBE_TEMP}"/approved_licenses.dump
 		fi
 	else
-		echo "${GO_PACKAGE}  ${LICENSE_NAME}  ${LICENSE_URL}" >> "${KUBE_TEMP}"notapproved_licenses.dump
+		echo "${GO_PACKAGE}  ${LICENSE_NAME}  ${LICENSE_URL}" >> "${KUBE_TEMP}"/notapproved_licenses.dump
 		packages_flagged+=("${GO_PACKAGE}")
 	fi
 done < "${KUBE_TEMP}"/licenses.csv
 awk '{ printf "%-100s : %-20s : %s\n", $1, $2, $3 }' "${KUBE_TEMP}"/approved_licenses.dump
 
 
-# cleanup
-git remote remove licenses
```

Comment:
fyi, this is already included in https://github.com/kubernetes/kubernetes/pull/114851 and in the merge queue

Reply:
ack, removed, thanks for following up!